components/alert/alert.css

```diff
@@ -0,0 +1,54 @@
+@import url("/index.css");
+@import url("sidebar.css");
+
+*{
+    margin:0;
+    padding: 0;
+}
+body{
+    background-image: none;
+}
+
+.alert-sidebar-main-container{
+    margin: 50px 0px 0px 0px;
+    display: flex;
+}
```

Comment:
use rem, please.

50px = 50/16 rem ~ 3.12rem

Reply:
Changed to rem. Please review it